packages/hawtio/tsconfig.json

```diff
@@ -20,6 +20,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "ignoreDeprecations": "5.0",
```

Comment:
It looks a bit alarming. Why do we need these?

Reply:
oh wait - it must've been unnecessary `git add`...